src/services/nitrous-api.js

```diff
@@ -1,5 +1,7 @@
 const axios = require("axios");
 
+const NITROUS_BASE_API_URL = "api-testnet.deezy.place";
```

Comment:
Can we load these as configs so we can set them in as netlfiy env vars eventually